index.js

```diff
@@ -4,6 +4,7 @@ const crypto = require('crypto');
 const urlSafeCharacters = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-._~'.split('');
 const numericCharacters = '0123456789'.split('');
 const distinguishableCharacters = 'CDEHKMPRTUWXY012458'.split('');
+const asciiPrintableCharacters = ' !"#$%&\'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~'.split('');
```

Comment:
Should we really include space? Since it's not actually visible. I'm honestly not sure, just feels a bit wrong. Maybe we could look at some prior art on what other solutions using printable ASCII characters did.